AntWars/Helper/Messages.cs

```diff
@@ -96,5 +96,10 @@ static class Messages {
         /// Sichtweite
         /// </summary>
         public static readonly string VIEWRANGE = "Sichtweite";
+
+        /// <summary>
+        /// {0} hat gewonnen! {1} hat weder genügend Ameisen noch genügend Geld.
```

Comment:
"hat weder genügend Ameisen noch genügend Geld." klingt nicht richtig, in welchem kontext nicht "genügend". sollte sowas sein wie hat keine ameisen und kein geld für neue.